wca-guidelines.md

```diff
@@ -222,6 +222,8 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-H><multiple-blindfolded><multipleblindfoldedsolving> Article H: Multiple Blindfolded Solving
 
+- H1a+) [ADDITION] The competitors may choose to bring 1 or 2 extra puzzles as backup in case of incidents found during the attempt (e.g. duplicate scrambled puzzles). Attempts higher than 6 puzzles are highly recommendated to bring extra puzzles.
```

Comment:
To be more in line with the rest of the text, I'd suggest to change "The competitors" to simply "Competitors" or "The competitor" (same as H1a)
Also not sure if the "1 or 2" is necessary. Competitors trying 40 cubes may want to have more backups, as the risk of incidents is higher.

Also, recommendated -> recommended